docs/release-notes/breaking-changes.md

```diff
@@ -21,7 +21,7 @@ Before you upgrade, carefully review the Elastic APM breaking changes and take t
 
 ## 9.0.0 [9-0-0]
 
-**Release date:** March 25, 2025
+**Release date:** April 2, 2025
```

Comment:
I took this from the docs-content repo, can you help me understand if it is correct?

Reply:
This aligns to changes made in https://github.com/elastic/apm-server/pull/16304/files#diff-8fcce13fe252c7ff12450ef97e1d8f138bd647138bd426f428c41bbe274ef139R22, which is still open. Maybe it worth not adding it here?

Reply:
Agh, completely missed thanks.
It's a no-op once the other is merged and this is updated/rebased.

Reply:
I'd remove the release date here, not usually shared in release notes afair.

Reply:
It's part of the template AFAICT, i am glad to remove but it's also present in previous versions' release notes

Reply:
nvm, it's not present in older version - removed

Reply:
Tech writers team asked to leave the release date, i'll revert last commit